src/test/java/th/ac/kmitl/it/soa/group3/EtaxBackendApplicationTests.java

```diff
@@ -0,0 +1,14 @@
+package th.ac.kmitl.it.soa.group3;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class EtaxBackendApplicationTests {
+
+	// TODO
```

Comment:
น้องช่วยเพิ่มด้วยครับ ว่า TODO ที่น้องต้องการที่ กำลังจะทำอะไร (TODO => to do หรือสิ่งที่จะทำครับ)

Reply:
คอมมิตใหม่แล้วครับ ขอโทษครับพอดีเบลอเลยลืมเขียน